packages/fiori-app-sub-generator/src/fiori-app-generator/transforms.ts

```diff
@@ -346,11 +346,12 @@ function getBaseAppConfig(
             eslint: project.enableEslint,
             typescript: project.enableTypeScript,
             sapux: project.sapux,
-            loadReuseLibs: !service.capService,
+            loadReuseLibs: !service.capService && !project.enableVirtualEndpoints,
             // Striclty speaking we should not need to guard here. If a template is not supported for OPA test generation then nothing should be generated.
             addTests: canGenerateTests(template.type),
             generateIndex: generateIndexHtml,
-            addAnnotations: entityRelatedConfig?.addFEOPAnnotations || entityRelatedConfig?.addLineItemAnnotations
+            addAnnotations: entityRelatedConfig?.addFEOPAnnotations || entityRelatedConfig?.addLineItemAnnotations,
+            useVirtualPreviewEndpoints: project.enableVirtualEndpoints
```

Comment:
Should be `enableVirtualEndpoints` ?

Reply:
`useVirtualPreviewEndpoints` is part of `AppOptions`, which is already released
It also does not have the same convention. I thought your comments were specific to the prompts and app gen project interface